statistics/update.go

```diff
@@ -315,6 +315,51 @@ func (h *Handle) dumpFeedbackToKV(fb *QueryFeedback) error {
 	return errors.Trace(err)
 }
 
+// UpdateStatsByLocalFeedback will update statistics by local feedback.
```

Comment:
how about this comment:
```go
// UpdateStatsByLocalFeedback will update statistics by the local feedback.
// Currently, we dump the feedback with the period of 10 minutes, which means
// it takes 10 minutes for a feedback to take effect. However, we can use the
// feedback locally on this tidb-server, so it could be used more timely.
```